### DISCUSSION TITLE
Loop restarting explorer with OOBE white screen flash before self-disabling

### DISCUSSION THREAD

**Discussion by lancekeltner7** [valinet/ExplorerPatcher#1493] [Q&A]
I have three machines I use explorer patcher on. 2 of them work great, but my primary work computer (win 11 enterprise via M365 E3, 22H2, build 22621.1702) will not work with explorer patcher, and I've been installing and trying every version put out this year.

When I install it, it restarts explorer, waits 1-2 seconds, pops up a white screen that looks like the start of the OOBE experience, and then explorer stops again. It loops that 3 times before the disable message from explorer patcher pops up.  Before the disable message was included in a recent version, it would keep looping explorer and the OOBE popup until you killed explorer.exe in task manager.

The OOBE popup does not show any text, so I'm not sure it's actually that, but that's what it resembles. I have a short video of it happening here:  https://imgur.com/a/ZJyI66i

I would appreciate any insight on where to look to fix this.

**Comment by pyrates999** [comment]
This was filed on May 26 and should be closed due to inactivity from the author.

**Reply by lancekeltner7** [reply]
I finally moved to StartAllBack to solve the issue.

W11 23h2 update did some to help with this, but the task bar is still not the way I would like it so I keep this application for the machine that doesn't like explorer patcher, and explorer patcher on the rest.
